Microsoft.Toolkit.Uwp.SampleApp/App.xaml

```diff
@@ -16,7 +16,7 @@
 				<ResourceDictionary Source="ms-appx:///Styles/Generic.xaml" />
 
 				<!-- WinUI -->
-				<not_wasm:XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
+				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
```

Comment:
```suggestion
				<not_wasm:ResourceDictionary>
					<ResourceDictionary.MergedDictionaries>
						<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
					</ResourceDictionary.MergedDictionaries>
				<not_wasm:ResourceDictionary/>
```

Reply:
Indeed, this was size related for Wasm. It probably still is required.